_assets/javascripts/lib/authentication.js

```diff
@@ -0,0 +1,39 @@
+
+let oktaConfig = {
+  clientId: CRDS.env.okta_client_id,
```

Comment:
Is there a reason we are declaring clientId and issuer using {site.XXX} on auth-required and using CRDS.env here?

Reply:
just goofed on not changing that file from {site.....} when I realized that didnt work. Refactored anyway.